src/engine/e2e_tests/process/processEndpoint.e2e.test.js

```diff
@@ -71,164 +70,168 @@ describe('Test process endpoints', () => {
+          expect(response.body).toStrictEqual({
+            definitionId: '_64552049-90bf-4f5b-96dd-e00747261755',
+            versions: [
+              {
+                bpmn: bpmnWithImage,
+                definitionName: 'With Image',
+                deploymentMethod: 'dynamic',
+                deploymentDate: expect.any(Number),
+                needs: {
+                  html: ['User_Task_1qjpbcl-1671026484009'],
+                  imports: [],
+                  images: [
+                    'Activity_08fwikp_image123e6803-63a8-4cf1-9596-2999fdd016a7.png',
+                    'User_Task_1qjpbcl_image72fe83de-2c44-4d1f-ae71-6b323bee7f1c.png',
+                  ],
+                },
+                version: 1671026484009,
+                versionName: 'Version 1',
+                versionDescription: 'Initial Version',
+                basedOnVersion: '1671024712832',
+              },
+            ],
+            instances: [],
+          });
+        });
+      });
+
+      describe('/process/{definitionId}/instance', () => {
+        let instanceId;
+        beforeAll(async () => {
+          const getResponse = await request.get('/process/definitionId/instance');
+          expect(getResponse.status).toBe(200);
+          expect(getResponse.body).toStrictEqual([]);
+          const postResponse = await request.post('/process/definitionId/versions/123/instance');
+          expect(postResponse.status).toBe(201);
+          ({ instanceId } = postResponse.body);
+          // allow everything to start correctly (the user task should have completely started)
+          await new Promise((res) => setTimeout(res, 500));
+        });
+        it('responds with statuscode 404 on GET requests with nonexisting definitionId', async () => {
+          const response = await request.get('/process/unknownDefinitionId/instance');
+          expect(response.status).toBe(404);
+        });
+        it('returns array containing information about all instances of a process on GET requests', async () => {
+          const getResponse = await request.get('/process/definitionId/instance');
+          expect(getResponse.status).toBe(200);
+          expect(getResponse.body).toStrictEqual([instanceId]);
+        });
+        describe('/process/{definitionId}/instance/{instanceId}', () => {
+          it('responds with an information object for the instance with the requested id on GET requests', async () => {
+            const response = await request.get(`/process/definitionId/instance/${instanceId}`);
+            expect(response.status).toBe(200);
+            expect(response.body).toHaveProperty('instanceState');
+            expect(response.body).toHaveProperty('processId');
+            expect(response.body).toHaveProperty('processInstanceId');
+            expect(response.body).toHaveProperty('tokens');
+            response.body.tokens.forEach((token) => {
+              // TODO: expect(token).toHaveProperty('state');
```

Comment:
What's the todo here? This property is tested below, are there more to add at some point or are these comments obsolete?